gulpfile.js

```diff
@@ -59,7 +53,7 @@ var config = {
   },
   folderDist: {
     base: 'dist',
-    css: 'dist',
+    css: 'dist/css',
```

Comment:
Is it fine? we do not want css files in root path?

Reply:
I think either way is the same. I had an issue with the html template being used for dist and for dev at the same time, the route in the html is hardcoded. Perhaps we don't need to have an html in the dist folder.

Reply:
@cbernat clarified me that it was different only for dist, so she is making it consistent. I understand now. Thanks!